proto/metadata.proto

```diff
@@ -316,7 +316,11 @@ message Metadata {
     // Device does not support telemetry path
     // /qos/interfaces/interface/input/virtual-output-queues/voq-interface/queues/queue/state/dropped-pkts.
     bool qos_voq_drop_counter_unsupported = 103;
-
+    // Device does not support tunnel interfaces state paths
+    bool tunnel_state_path_unsupported = 104;
+    // Device does not support tunnel interfaces source and destination address config paths
+    bool tunnel_config_path_unsupported = 105;
```

Comment:
Please raise a deviation bug similar to b/299545397

Reply:
updated bug